packages/netglade_utils/lib/src/testing/mock_extensions.dart

```diff
@@ -0,0 +1,77 @@
+// ignore_for_file: prefer-match-file-name
+
+import 'dart:async';
+
+import 'package:mocktail/mocktail.dart';
+import 'package:netglade_utils/src/result/result.dart';
+import 'package:netglade_utils/src/typedefs/typedefs.dart';
+
+extension FutureVoidAnswer on When<Future<void>> {
+  void thenAnswerWithVoid() => thenAnswer((_) async => <void>{});
+}
+
+extension VoidAnswer on When<void> {
+  void thenAnswerWithVoid() => thenAnswer((_) => <void>{});
+}
+
+extension GenericAnswer<T> on When<Future<T>> {
+  void thenAnswerWith(T value) => thenAnswer((_) => Future<T>.value(value));
+}
+
+extension StreamedAnswer<T> on When<Stream<T>> {
+  void thenEmitValues(List<T> values) => thenAnswer((_) => Stream.fromIterable(values));
+}
+
+extension VerificationResultEx on VerificationResult {
+  void calledOnce() => called(1);
+}
+
+// TODO(utils): add when we will add api service
```

Comment:
Remove this comment? ApiService will not be added directly in this package anyway